lib/utils.pm

```diff
@@ -392,7 +392,7 @@ sub zypper_call {
 
 sub fully_patch_system {
     # first run, possible update of packager -- exit code 103
-    zypper_call('patch --with-interactive -l', exitcode => [0, 102, 103], timeout => 1500);
+    zypper_call('patch --with-interactive -l', exitcode => [0, 102, 103], timeout => 3000);
```

Comment:
sorry not so keen in upgrade process with zypper but it looks really strange that you need to run same command twice . 
In any case could it be that in certain circumstances first command trigger full upgrade ? This would explain failure in different way . Just saying that maybe you need to increase it even to 6000 ?

Reply:
Run same command twice instead of one update command indeed strange, but this not related with my current change. @okurz Maybe you can give some clue?
I have rerun the failed cases, it will spend about 1200s, so that's why i suppose double the current value 1500 to 3000 is enough. If you think 6000 is better, reopen the comments:)